Drivers/Threshold_class.py

```diff
@@ -0,0 +1,77 @@
+import os
+from dotenv import load_dotenv
+
+class FTP:
```

Comment:
Recommend changing the name of this class to Threshold or something other than FTP, as its not an FTP class object.